src/types/annotation.ts

```diff
@@ -14,14 +14,14 @@ export interface AnnotationStream {
 
 export interface GetAnnotationPayload {
   stream?: string
-  start?: string
-  end?: string
+  start?: number
+  end?: number
   stickers?: any
 }
 
 export interface DeleteAnnotation {
   stream: string
-  start: string
-  end: string
+  start: number
+  end: number
```

Comment:
i think you shot yourself in the foot here 😄 if you change these back to `string` (which they should be, these are what we send across the wire) and remove the `eslint-ignore` and `ts-ignore`, you should get a clean build.